app/tests/client/db_client.py

```diff
@@ -0,0 +1,151 @@
+import asyncio
+import time
+from functools import partial
+from operator import le
+from os import listdir
+from os.path import isdir, isfile, join
+from types import TracebackType
+from typing import Any, Iterable, Optional, Type
+
+import aioodbc
+from databases.core import DatabaseURL
+from loguru import logger
+from toolz import compose
+
+from app.tests.settings import test_settings
+from app.tests.utils.utils import (
+    Location,
+    get_number_in_str,
+    read_content_from_file,
+)
+
+
+class MSSQLClient:
```

Comment:
what is the difference between this one and our normal client at database/database.py?

Reply:
Functionally not much. I originally was using the normal client, however after discussing with Alvin we thought a separate client for tests would be better because we want to be doing testing for the normal client. Thought it would be a little unorthodox to be using that will be tested anyway, but open to suggestions